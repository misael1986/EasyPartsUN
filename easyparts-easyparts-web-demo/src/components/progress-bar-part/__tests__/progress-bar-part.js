import React from "react";
import renderer from "react-test-renderer";
import ProgressBarPart from "../progress-bar-part";

describe("ProgressBarPart", () => {
  it("hasn't changed unexpectedly", () => {
    const part = { imagen: "", nombre: "" };
    const tree = renderer
      .create(
        <ProgressBarPart
          type="cpu"
          part={part}
          name="Procesador"
          returnToPart={() => {}}
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
