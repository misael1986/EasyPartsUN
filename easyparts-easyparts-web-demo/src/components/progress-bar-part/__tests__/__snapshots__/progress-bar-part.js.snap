// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBarPart hasn't changed unexpectedly 1`] = `
<div
  className="ProgressBarPart"
  onClick={[Function]}
  onPointerLeave={[Function]}
>
  <div
    className="PartSelected"
    onPointerEnter={[Function]}
  >
    <p
      className="PartSelectedTitle"
    >
      Procesador
    </p>
    <img
      alt="Imagen del Producto"
      className="PartSelectedImage"
      src=""
    />
    <p
      className="PartSelectedDesc"
    >
      
    </p>
  </div>
</div>
`;
